mutagen-core/src/mutator/mutator_binop_add.rs

```diff
@@ -0,0 +1,95 @@
+//! Mutator for binary operation `+`.
+
+use std::ops::Add;
+
+use syn::spanned::Spanned;
+use syn::{parse_quote, BinOp, Expr, ExprBinary};
+
+use crate::transform_info::SharedTransformInfo;
+use crate::transformer::ExprTransformerOutput;
+use crate::Mutation;
+
+use crate::optimistic::AddToSub;
+use crate::MutagenRuntimeConfig;
+
+pub struct MutatorBinopAdd {}
+
+impl MutatorBinopAdd {
+    pub fn run<L: Add<R>, R>(
+        mutator_id: u32,
+        left: L,
+        right: R,
+        runtime: MutagenRuntimeConfig,
+    ) -> <L as Add<R>>::Output {
+        if runtime.mutation_id != mutator_id {
```

Comment:
Do I understand right that a Mutator can carry out only a single mutation? The current design allows a single mutation to add several code paths.

Reply:
It was always the idea to allow multiple mutations per mutator and I implemented this for the mutator `lit_int` in the newest commit. This required more changes than expected.

Reply:
:+1: